tutorials/conventions.ipynb

```diff
@@ -1,17 +1,26 @@
 {
```

Comment:
<p>I like that you spell out the equation. Could you add Hooke's law (for instance, after "body force. ": </p><p>"Usually the stress tensor is related to displacement through Hooke's law and the strain-displacement relation for small deformations:</p><p>sigma(u) = C : \epsilon(u) = C / 2 : (\nabla u + \nabla u T)".</p><p>The implementation details should perhaps rather be specified elsewhere. If we put it in the actual implementation, we could refer to this notebook for the convention/equations.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pmgbergen/porepy/pull/904/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>Thanks for the input about Hooke's law, it is now included, I removed the implementation detail section, and the (-1) multiplication is already mentioned in the momentum_balance_equation method itself. So perhaps that is sufficient?&nbsp;</p>